### PR TITLE
Support snake_case in njk files.

### PR DESCRIPTION
**Pull request by hwanseung**
If interfaces has CamelCase name in idl files, it have to change to
snake_case for cpp files name and Include guards. So we should support
to snake_case filter in nunjucks files.

ISSUE=#94

### CODE REVIEW

File: generator/main.ts [hunk 1]
```diff
@@ -20,6 +20,7 @@ import * as mkdirp from 'mkdirp';
 import * as nunjucks from 'nunjucks';
 import * as path from 'path';
 import * as webidl from 'webidl2';
+import snakeCase = require('snake-case');
```
Comment (romandev) [inline generator/main.ts:23]: Please use import instead of require :)

**Review by romandev — COMMENTED** [review]
Thank you for this change.

File: generator/main.ts (hunk 1 above)
Comment (hwanseung) [reply generator/main.ts:23]: unfortunately, this snake_case module is ```export = and import = require()``` case.
so we have to ```require``` keyword to use this module.
(refer to https://www.typescriptlang.org/docs/handbook/modules.html
and https://github.com/blakeembrey/snake-case/blob/master/snake-case.d.ts#L3)

do we have to use the other modules for snake_case module to void this case?

Comment (romandev) [reply generator/main.ts:23]: Ack

**Pull request merged**